src/app/app.component.scss

```diff
@@ -30,5 +30,121 @@
 }
 
 .mdc-list-item.active {
-  background: rgba(255,255,255, 0.08);
+  background: rgba(156, 39, 176, .15)
 }
+
+header {
+  display: block;
+  overflow: hidden;
+  position: relative;
+  padding-bottom: 2em;
+}
+
+.container {
+  width: 320px;
+
+  padding-bottom: 1em;
+}
+
+.bg {
+  border-bottom: 8px solid #9c27b0;
+}
+
+.bio:hover>.desc {
+  cursor: pointer;
+  opacity: 1;
+}
+
+.avatarcontainer {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  margin: auto;
+  width: 70px;
+  display: block;
+}
+
+.avatarcontainer:hover>.hover {
+  opacity: 1;
+}
+
+.avatar {
+  width: 100%;
+  border: 8px solid #9c27b0;
+  -webkit-border-radius: 50%;
```

Comment:
We don't care about old compatibility, most browsers are now up-to-date on the latest standards and especially border-radius gotta be supported for very long time now?